driver-core/src/main/com/mongodb/client/model/BulkWriteOptions.java

```diff
@@ -76,11 +79,50 @@ public BulkWriteOptions bypassDocumentValidation(@Nullable final Boolean bypassD
         return this;
     }
 
+    /**
+     * Returns the comment to send with the query. The default is not to include a comment with the query.
+     *
+     * @return the comment
+     * @since 4.6
+     * @mongodb.server.release 4.4
+     */
+    @Nullable
+    public BsonValue getComment() {
+        return comment;
+    }
+
+    /**
+     * Sets the comment for this operation. A null value means no comment is set.
+     *
+     * @param comment the comment
+     * @return this
+     * @since 4.6
+     * @mongodb.server.release 4.4
+     */
+    public BulkWriteOptions comment(@Nullable final String comment) {
```

Comment:
Opted to add both `comment(String)` and `comment(BsonValue)` throughout - to keep the api's the consistent. Also, I took the opinion that most users won't be familiar with BsonValues.